package.json

```diff
@@ -13,7 +13,7 @@
     "typescript-eslint": "^7.13.1",
     "vitest": "^1.6.0"
   },
-  "name": "cortex-sdk",
+  "name": "@cortexclick/cortex",
```

Comment:
I'm worried this package name is confusing. Should it be like `@cortexclick/js` or `@cortexclick/sdk`? `cortex` does not really describe this package.

Reply:
People do all sorts of differnt things with package names. I prefer to stick to something branded for the SDK. Similar to openAI and pulumi: 

- OpenAI's SDK is just called `openai` https://www.npmjs.com/package/openai
- Pulumi's SDK is `@pulumi/pulumi` https://www.npmjs.com/package/@pulumi/pulumi

But others do just use sdk: 

- Anthropic: https://www.npmjs.com/package/@anthropic-ai/sdk

Reply:
And `cortex` is the key noun for our offering. A `cortex` is a customizable AI agent for marketing/sales automation that let's you operate over your enterprise data. 

We will have some work to do one the marketing and brand awareness side, but I think we should lean into it and not away from it.

Reply:
Why not `cortex-sdk`?